### ISSUE TITLE
New version of cursor (0.44) breaks with dev containers

### ISSUE THREAD

**Issue by JamesWordie** [getcursor/cursor#2501]
The newest release or version of cursor appears to break dev containers and there ability to open locally. 

I updated this afternoon to 0.44 and it now fails/errors with the below;

![image](https://github.com/user-attachments/assets/58a2f5ae-b104-4579-8366-94449f56cfb7)

I am on cursor version: 0.44
Docker Version: 27.2.0
dev container extension: latest

Note: deleting the old container and images in docker also doesn't resolve the problem

Are you able to advise or suggest ETA on a fix?

NOTE: We will try our best to check Github Issues, but [forum.cursor.com](https://forum.cursor.com/) is our main site for bug reports / user feedback. We'd recommend you post there for the fastest response.

**Comment by JamesWordie** [comment]
And for reference dev-container json looks like;

```
{
	"name": "bg-ext-research",
	"build": {
		"dockerfile": "./Dockerfile",
		"context": "../"
	},
	"customizations": {
        "vscode": {
            "extensions": [
                "ms-python.python",
				"ms-python.debugpy",
				"ms-toolsai.jupyter"
            ],
			"settings": {
				"python.pythonPath": "/usr/local/bin/python"
			}
        }
    },
    "initializeCommand": "(az account list-locations -o none || az login) && az acr login -n devreg && echo 'logged in to devreg successful'",
    "postCreateCommand": "mkdir -p /root/cert && chown root /root/cert && cp /etc/ssl/certs/ca-certificates.crt /root/cert/ca-certificates.crt && mkdir /run/secrets",
	"postStartCommand": "git config --global core.autocrlf true",
	"containerEnv": { 
		"REQUESTS_CA_BUNDLE": "/etc/ssl/certs/ca-certificates.crt",
		"NODE_EXTRA_CA_CERTS": "/etc/ssl/certs/ca-certificates.crt"
	},
	"runArgs": ["--env-file",".devcontainer/.env","--name","bg-ext-research"],
    "remoteUser": "root"
}
```

**Comment by JamesWordie** [comment]
Fixed with version 0.44.3
